modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -27,33 +27,74 @@ just like your other ownCloud files and folders.
 Please see the details about the xref:enterprise/external_storage/enterprise_only_auth.adoc[Enterprise-Only Authentication Options]
 
-WND has the ability to trigger change notifications to users based on changes for files and folders. 
+Listen to change information triggered by the backend::
+Native Windows File Servers provide the ability to proactively send change notifications made to files and folders somewhere in a share. This mechanism then updates the ownCloud database and provides the changes made to accessing users. Users do not need to manually check for changes in all possible locations of their mount. Changes processed are also propagated to sync clients automatically.
 
-With enhanced ACL support, both SMB and WND evaluate the file attributes (whether the file is hidden or read-only) to decide what ownCloud permissions should the file or folder have in ownCloud. On top of this, WND can also evaluate the ACLs by using the "ocLdapPermissionManager" in the mount point configuration. This will bring more accurate permissions to ownCloud, specially if each user could have different permissions over the files in Windows.
+Enhanced ACL support::
+With enhanced ACL support, both SMB and WND evaluate the file attributes (whether the file is hidden or read-only) to decide what ownCloud permissions should the file or folder have in ownCloud. On top of this, WND can also evaluate the ACLs by using the `ocLdapPermissionManager` in the mount point configuration. This will bring more accurate permissions to ownCloud, specially if each user could have different permissions over the files in Windows.
 
-Mounts to a Windows or Samba file server are labeled with a little four-pane Windows-style icon, and the left pane of your Files page includes a Windows Network Drive filter. Figure 1 shows a new Windows Network Drive share marked with red warning which indicates that ownCloud cannot connect to the share. The reason is that it may require the user to login, or it is not available, or there is an error in the configuration.
+Collaborative WND (CWND)::
+Compared to a standard WND mountpoint, a collaborative WND mount offers enhanced features. In a CWND, each user shares the same ownCloud internal informations for files and folders based on its internal identification (file_id). This means that _comments_ and _tags_ can be shared with all users accessing files and folders of this mount without the need that users must be member of the mount from ownCloud point of view. A CWND can only be set by an admin in menu:Settings[Admin > Storage] but not in the users section. With CWND, you lose the benefit of sharing a mount and therefore sharing e.g. comments and tags, but it removes the drawback that you only have one accessing user, the sharer. With CWND, all accessing users have their own access with their credentials but share additional information with other users accessing the same data. See the table below to compare the differences.
```

Comment:
```suggestion
Compared to a standard WND mountpoint, a collaborative WND mount offers enhanced features. In a CWND, each user shares the same ownCloud internal information for files and folders based on its internal identification (file_id). This means that _comments_ and _tags_ can be shared with all users accessing files and folders of this mount without the need that users must be members of the mount from an ownCloud point of view. A CWND can only be set by an admin in menu:Settings[Admin > Storage] but not in the users section. With CWND, you lose the benefit of sharing a mount and therefore sharing e.g. comments and tags, but it removes the drawback that you only have one accessing user, the sharer. With CWND, all accessing users have their own access with their credentials but share additional information with other users accessing the same data. See the table below to compare the differences.
```

Reply:
"This means that _comments_ and _tags_ can be shared with all users" and "With CWND, you lose the benefit of sharing a mount and therefore sharing e.g. comments and tags"
I am confused - are comments and tags shared or not? In what situations?

Reply:
You are right, needs rewriting.
I need an info from @jvillafanez , he got the question already. When I get it, I will fix this.

Reply:
This part completely confuses me: "With CWND, you lose the benefit of sharing a mount" -- I thought the collaborative feature was about improving handling of shared resources. (but forgive me, I haven't read the rest yet.)
The explanation that follows introduces more topics: We learn that "only .. one accessing user" is a drawback. That seems to contradict the introduction of one service user to rule them all -- which should be more a solution than a problem.

Reply:
text is fixed